client/coral-admin/src/routes/Moderation/containers/Moderation.js

```diff
@@ -208,7 +212,7 @@ const withModQueueQuery = withQuery(gql`
     }) {
       ...CoralAdmin_Moderation_CommentConnection
     }
-    assets: assets {
+    assets: assets(query: {}) {
```

Comment:
@okbel This also breaks, because the AssetsQuery type has a default limit of 10 ;-) 
I think you can just use `assets {`.